packages/drivers/replay-driver/src/replayDocumentService.ts

```diff
@@ -41,12 +42,22 @@ export class ReplayDocumentService
 			await documentService.connectToDeltaStorage(),
 			controller,
 		);
-		return new ReplayDocumentService(controller, deltaConnection);
+		const maybeDriverCompatDetails = documentService as FluidObject<ILayerCompatDetails>;
```

Comment:
What kind of documentService might this be?  I couldn't trace back any code that actually uses replay driver, I'm sure I missed it, but not familiar with how it works.